server/repository/src/migrations/v1_07_00/name_store_join_add_name_link_id.rs

```diff
@@ -35,6 +35,20 @@ pub(crate) fn migrate(connection: &StorageConnection) -> anyhow::Result<()> {
             ADD COLUMN name_link_id TEXT NOT NULL REFERENCES name_link (id) DEFAULT 'temp_for_migration'; 
             UPDATE name_store_join SET name_link_id = name_id;
             PRAGMA foreign_keys = ON;
+
+        CREATE TRIGGER name_store_join_insert_trigger
+          AFTER INSERT ON name_store_join
+          BEGIN
+            INSERT INTO changelog (table_name, record_id, row_action, store_id, is_sync_update)
+              VALUES ('name_store_join', NEW.id, 'UPSERT', NEW.store_id, NEW.is_sync_update);
```

Comment:
Does changelog no longer do `name_id`? The postgres equivalent does right? (i'm looking at develop tho)

https://github.com/msupply-foundation/open-msupply/blob/e8de66c1eecd7e6e7c6d5c394ab541e3a7426e6d/server/repository/src/migrations/v1_07_00/name_store_join_add_name_link_id.rs#L16-L24

Reply:
postgres one does... but the sql one didn't when it was first created? Not sure the reason? Can add tho